tests/functional/change_password.js

```diff
@@ -1,74 +1,70 @@
+'use strict';
```

Comment:
Noticed some test files have the `use strict` and some don't. Should it be put on all?

Reply:
cleaned these up!